File: py/IEC61968/Common/Author.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from IEC61968.Common.ActivityRecord import ActivityRecord
from IEC61968.Common.Document import Document
from IEC61968.Common.DocumentPersonRole import DocumentPersonRole


class Author(DocumentPersonRole):
    
    def __init__(self):
        super().__init__()
        self.documents = Document()
        self.activity_records = ActivityRecord()

File: java/IEC61968/Common/Author.java
package IEC61968.Common;


/**
 * Person who created document or activity record.
 * @author T. Kostic
 * @version 1.0
 * @created 25-Dec-2023 8:45:19 PM
 */
public class Author extends DocumentPersonRole {

	/**
	 * All documents of this this author.
	 */
	public Document Documents;
	/**
	 * All activity records with this author.
	 */
	public ActivityRecord ActivityRecords;

	public Author(){

	}
}//end Author